src/main/drivers/opticalflow/opticalflow.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <stdint.h>
+
+#define OPTICALFLOW_OUT_OF_RANGE        -1
+#define OPTICALFLOW_HARDWARE_FAILURE    -2
+#define OPTICALFLOW_NO_NEW_DATA         -3
+
+#define OPTICALFLOW_HARDWARE_TIMEOUT_MS  100
+
+typedef struct opticalflowRates_s {
+    int32_t X;
+    int32_t Y;
+} opticalflowRates_t;
```

Comment:
THis will force `#include` of driver header info public `sensor` one. That is considerable namespace pollution. use `vector2_t`